includes/modules/payment/paypalwpp.php

```diff
@@ -280,8 +280,8 @@ function update_status() {
       $this->zcLog('update_status', 'Module disabled because purchase price (' . $order->info['total'] . ') exceeds PayPal-imposed maximum limit of 1000000 JPY.');
     }
     // module cannot be used for purchase > $10,000 USD equiv
-    $order_amount = $this->calc_order_amount($order->info['total'], 'USD', false);
-    if ($order_amount > 10000) {
+    require_once DIR_FS_CATALOG . DIR_WS_MODULES . 'payment/paypal/paypal_currency_check.php';
+    if (!paypalUSDCheck($order->info['total'])) {
```

Comment:
For this one, I'd use `=== false` instead of `!`.

Reply:
Yes agreed. It is clearer and easier to see.